queries/cedar/highlights.scm

```diff
@@ -0,0 +1,62 @@
+; Function calls
+(call_expression
+  function: (selector_expression
+    field: (field_identifier) @function.method))
+
+[
+  "contains"
+  "containsAll"
+] @function
+
+; Identifiers
+[
+  (entity)
+  (principal)
+  (action)
+  (resource)
+  (context)
+  (identifier)
+  (annotation)
+] @variable
```

Comment:
`(annotation)` should be `@attribute`.
I'm sure the others (barring `(identifier)`) can also be something else.

Reply:
Thanks, I've fixed `annotation` to be `@attribute`. For the others, `principal`, `action`, `resource` and `context` are "well-known variables" that are always defined for a particular policy evaluation. Seems like `@variable.builtin` could be a good fit?

Reply:
Whatever makes sense for the grammar according to [CONTRIBUTING.md](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#highlights).